### PR TITLE
Add basic SDK functionality and README

### PR DESCRIPTION
**Pull request by jzhu-eth**

### CODE REVIEW

File: get.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+// Get retrieves data from the IPFS network using the given CID.
+func (c *Client) Get(cid string) ([]byte, error) {
```
Comment (maxkaneco) [inline get.go:12]: Maybe use cid.Cid type? or validate the string?

File: get.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+		return nil, err
+	}
+
+	resp, err := c.hc.Do(req)
+	if err != nil {
+		return nil, errors.WithMessage(err, "call gateway API")
+	}
+
+	defer resp.Body.Close()
+	return ioutil.ReadAll(resp.Body)
```
Comment (maxkaneco) [inline get.go:36]: Don't we need to handle the redirection?

Comment (jzhu-eth) [reply get.go:36]: The Golang HTTP will handle the redirection 10 times in a call.

Comment (maxkaneco) [reply get.go:36]: good to know!

**Review by maxkaneco — APPROVED** [review]

**Pull request merged**